plugins/dashboard/drng_livefeed.go

```diff
@@ -42,11 +42,14 @@ func runDrngLiveFeed() {
 	})
 
 	daemon.BackgroundWorker("Dashboard[DRNGUpdater]", func(shutdownSignal <-chan struct{}) {
-		drng.Instance.Events.Randomness.Attach(notifyNewRandomness)
+		if !drng.Enabled() {
+			return
+		}
+		drng.Instance().Events.Randomness.Attach(notifyNewRandomness)
 		drngLiveFeedWorkerPool.Start()
 		<-shutdownSignal
 		log.Info("Stopping Dashboard[DRNGUpdater] ...")
-		drng.Instance.Events.Randomness.Detach(notifyNewRandomness)
+		drng.Instance().Events.Randomness.Detach(notifyNewRandomness)
 		newMsgRateLimiter.Stop()
```

Comment:
The stop should be deferred, right after `NewTicker`.
Also consider deferring the other statements to assure cleanup in case of panic.